.github/workflows/e2e_openssl.yml

```
@@ -60,4 +61,11 @@ jobs:
           just coordinator initializer openssl port-forwarder node-installer
       - name: E2E Test
         run: |
-          nix shell .#contrast.e2e --command openssl.test -test.v workspace/just.containerlookup
+          nix shell .#contrast.e2e --command openssl.test -test.v \
+            --image-replacements workspace/just.containerlookup \
+            --namespace-file workspace/e2e.namespace \
+            --skip-undeploy=${{ inputs.skip-undeploy }}
+      - name: Cleanup
+        if: cancelled() && inputs.skip-undeploy != true
```

```suggestion
        if: cancelled() && inputs.skip-undeploy
```

Now that we have a bool, this should be enough, right?

---

Sorry, forgot the negation, but you know what I mean :)